config.go

```diff
@@ -541,3 +551,112 @@ type VersionMistmatchError string
 func (e VersionMistmatchError) Error() string {
 	return string(e)
 }
+
+// Restart re-run the app with all the arguments passed
+func (config *TGFConfig) Restart() int {
+	cmd := exec.Command(os.Args[0], os.Args[1:]...)
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	if err := cmd.Run(); err != nil {
+		printError("Error on restart: %v", err)
+		return 1
+	}
+	return 0
+}
+
+// Debug print debug information with formatting string
+func (config *TGFConfig) Debug(format string, args ...interface{}) {
```

Comment:
I may have said `Debug` but it really isn't clear what it should do from outside the implementation. Call it `LogDebug` at least